repo/packages/K/kubernetes/1/config.json

```diff
@@ -0,0 +1,195 @@
+{
+  "properties": {
+    "kubernetes": {
+      "description": "Kubernetes specific configuration properties",
+      "properties": {
+        "app-id": {
+          "default": "kubernetes",
+          "description": "Application ID.",
+          "type": "string",
+          "pattern": "^(?:[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)$"
+        },
+        "docker-image": {
+          "default": "mesosphere/kubernetes:v1.0.1-v0.6.0-alpha",
+          "description": "Docker image used to launch the Kubernetes framework on DCOS.",
+          "type": "string"
+        },
+        "dns-suffix": {
+          "default": ".marathon.mesos",
+          "description": "This value is appended to the app-id value to form the canonical DNS name for the Kubernetes components.",
+          "type": "string",
+          "pattern": "^(?:\\.[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)+$"
+        },
+        "enable-dns": {
+          "default": "1",
+          "description": "Set to 1 to enable Kubernetes local DNS services, allows pods to locate services by name.",
+          "type": "string",
+          "pattern": "^1?$"
+        },
+        "framework-name": {
+          "default": "kubernetes",
+          "description": "Human-readable name to use when registering with the Mesos master.",
+          "type": "string",
+          "pattern": "^(?:[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)$"
+        },
+        "kube-dns-domain": {
+          "default": "kubernetes.local",
```

Comment:
looks like the k8s default is now `cluster.local`; we should probably strive for consistency here